server/serializers.py

```diff
@@ -32,9 +33,14 @@ class Meta:
 
 
 class ModuleSerializer(serializers.ModelSerializer):
+    help_url = serializers.SerializerMethodField()
+
+    def get_help_url(self, obj):
+        return obj.help_url if obj.help_url else KB_ROOT_URL
```

Comment:
Don't we have to add KB_ROOT_URL to help_url? I am envisioning that the URL in the json will be relative, so just 'add-data/add-from-url'. The idea is that we can seamlessly rebase the KB links, if e.g. we move off Intercom